src/models/SearchResultsBib.ts

```diff
@@ -0,0 +1,81 @@
+import type { SearchResult } from "../types/searchTypes"
+import type { ElectronicResource } from "../types/bibTypes"
+
+export default class SearchResultsBib {
+  id: string
+  title: string
+  yearPublished?: string
+  materialType?: string
+  publicationStatement?: string
+  eResources?: ElectronicResource[]
```

Comment:
Can this be `electronicResources`? I think it would be a good idea to match these property names exactly to the discovery api properties.

Reply:
Good call! Renamed to electronicResources